server/src/main/java/org/opensearch/action/admin/cluster/health/ClusterHealthRequest.java

```diff
@@ -90,6 +94,12 @@ public ClusterHealthRequest(StreamInput in) throws IOException {
         }
         waitForNoInitializingShards = in.readBoolean();
         indicesOptions = IndicesOptions.readIndicesOptions(in);
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            if (in.readBoolean()) {
+                awarenessAttribute = in.readString();
+            }
```

Comment:
nit : this can be simplified by `in.readOptionalString` and `out.writeOptionalString`